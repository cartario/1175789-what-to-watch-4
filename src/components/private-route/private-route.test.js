import React from "react";
import renderer from "react-test-renderer";
import PrivateRoute from "./private-route.jsx";
import {Provider} from 'react-redux';
import configureStore from 'redux-mock-store';
import {history} from "../../history.js";
import {Router} from 'react-router-dom';

const mockStore = configureStore([]);
const store = mockStore({
  USER: {

  },
});

describe(`Private-route-test`, () => {
  it(`should render private-route`, () => {

    const tree = renderer
      .create(
          <Router history = {history}>
            <Provider store = {store}>
              <PrivateRoute
                authorizationStatus = {``}
              />
            </Provider>
          </Router>
      )
      .toJSON();

    expect(tree).toMatchSnapshot();
  });
});
