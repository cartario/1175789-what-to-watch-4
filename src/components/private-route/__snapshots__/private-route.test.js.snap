// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Private-route-test should render private-route 1`] = `null`;
